src/linker/steps/dev/python_pandas/Dockerfile

```diff
@@ -1,12 +1,13 @@
 # https://hub.docker.com/layers/library/python/3.10/images/sha256-1c26c25390307b64e8ff73e7edf34b4fbeac59d41da41c08da28dc316a721899?context=explore
 FROM python@sha256:1c26c25390307b64e8ff73e7edf34b4fbeac59d41da41c08da28dc316a721899
 RUN mkdir -p /input_data
+RUN mkdir -p /extra_implementation_specific_input_data
```

Comment:
Not for that b/c it's not a _required_ bound dictionary. So this will make the folder in the container but allow the container to run even if you don't bind something to it.

Reply:
Actually, I'm kind of confused what VOLUME actually does in a Dockerfile. I need to research more but frankly I don't think it matters much whether we make the call here or not since we're explicitly binding in the run command.

Reply:
VOLUME

- Has no effect in Singularity, only Docker.
- Makes a directory _always_ bind somewhere on the host file system. Not having VOLUME does not prevent you from binding the directory.